sdat2img.py

```diff
@@ -74,7 +74,7 @@ def parse_transfer_list_file(path):
     elif version == 3:
         print('Android Marshmallow 6.x detected!\n')
     elif version == 4:
-        print('Android Nougat 7.x / Oreo 8.x detected!\n')
+        print('Android Nougat 7.x / Oreo 8.x / Pie 9.x / Q 10.x detected!\n')
```

Comment:
It should be `Pie 9 / Q 10`

Reply:
ok

Reply:
fixed it, android p and q tested working